torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -787,6 +787,23 @@ const std::vector<std::string> functions = {
 
             return result, backward
 
+        def logit(self, eps: Optional[float]):
```

Comment:
why are you defining symbolic gradient? This would cause scripted module run this implementation instead of built-in backward, and without fuser this will be much slower. And I'm not sure if fuser (which of them?) is able to efficiently fuse this backward.

Reply:
I don't know why if I didn't add it here, there will be an error that autodiff node number didn't match. So I added it here to test that.

Reply:
Which test was failing?